### DISCUSSION TITLE
Update to Home Assistant Add-on to latest version?

### DISCUSSION THREAD

**Discussion by tmason818** [jshank/bwalink#28] [Q&A] [answered]
I realized bwalink has been updated to 2024.02 and I'm still running 2023.6.

What's the methodology for updating the Home Assistant Add-on to 2024.02?  I tried just amending the original bwalink folder I had with "_OLD" and putting the new version files in its place and to my surprise I was prompted to update in the Add-ons section of Home Assistant. But when I clicked update it failed to update. 

I'm probably overthinking this but I don't want to mess things up and erase all my entities or end up with BOTH 2024.02 and 2023.6 somehow  😅

**Comment by jshank** [comment, marked as answer]
Check https://github.com/jshank/bwalink/issues/16#issuecomment-2302502252 for details on updating.

**Comment by tmason818** [comment]
Ah I see.  Due to the circular nature of Github I sometimes have trouble knowing "what's where" so to speak.

I did the rebuild it did take some time to run a job of some sort. The Home Assistant Add-on is still showing 2023.6 but _I think_ that is to be expected. Thanks!

**Comment by jshank** [comment]
Yep. This project is always a bit of a weird one. I'm not the author of the [balboa_worldwide_app](https://github.com/ccutrer/balboa_worldwide_app) which is really where all of the magic happens. I created this project originally to make it more accessible to Home Assistant through a Docker deployment that contained Ruby, BWA and some of my workarounds to make it more stable. As with most projects, folks have come along and significantly improved the implementation beyond my capabilities.
